messages/teleporter/message_manager.go

```diff
@@ -23,20 +23,18 @@ import (
 	"go.uber.org/zap"
 )
 
-const (
-	teleporterMessageCacheSize = 100
-)
-
 type messageManager struct {
-	messageConfig   Config
-	protocolAddress common.Address
-
-	// We parse teleporter messages in ShouldSendMessage, cache them to be reused in SendMessage
-	// The cache is keyed by the Warp message ID, NOT the Teleporter message ID
-	teleporterMessageCache *cache.LRU[ids.ID, *teleportermessenger.TeleporterMessage]
-	destinationClients     map[ids.ID]vms.DestinationClient
+	messageConfig      Config
+	protocolAddress    common.Address
+	destinationClients map[ids.ID]vms.DestinationClient
+	logger             logging.Logger
+}
 
-	logger logging.Logger
+type messageHandler struct {
```

Comment:
Can we move `messageHandler` out to its own file? I found it a bit hard to find under `message_manager.go`